misc/alert_rules.json

```diff
@@ -313,30 +313,6 @@
     "rule": "devices.os = \"panos\" && sensors.sensor_type = \"panSysHAState\" && sensors.sensor_current = \"1\" && sensors.sensor_prev = \"2\"",
     "name": "Palo Alto Networks passive firewall changed to active"
   },
-  {
-    "rule": "sensors.sensor_current ~ \"[2|6]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10893.1.20.130.15.1.4.1\"",
-    "name": "Dell Server Raid Battery Failed/Degraded"
-  },
-  {
-    "rule": "sensors.sensor_current = \"2\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10892.1.1100.32.1.5\"",
-    "name": "Dell Server CPU Status Critical"
-  },
-  {
-    "rule": "sensors.sensor_current ~ \"[2|6]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10893.1.20.130.1.1.5\"",
-    "name": "Dell Server Disk Controller State Failed/Degraded"
-  },
-  {
-    "rule": "sensors.sensor_current ~ \"[2|5]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10893.1.20.130.4.1.4\"",
-    "name": "Dell Server Disk Array State Failed/Degraded"
-  },
-  {
-    "rule": "sensors.sensor_current ~ \"[5|6]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10892.1.600.12.1.5\"",
-    "name": "Dell Server PSU State Critical/NonRecvoverable"
-  },
-  {
-    "rule": "sensors.sensor_current ~ \"[2|6]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10893.1.20.140.1.1.4\"",
-    "name": "Dell Server Virtual Disk Failed/Degraded"
-  },
```

Comment:
I agree that these should probably be removed or converted to use the more general state sensor rule. The whole Dell-specific code should probably be upgraded.